cmd/cmd.go

```diff
@@ -122,8 +151,10 @@ func (o *RootCommandOption) Run(streams IOStreams) error {
 	var errCnt int
 	for _, f := range o.files {
 		base := filepath.Base(f)
```

Comment:
You also can define`base` var inside if block, if so the variable scope could be narrowed little more.